areas-of-interest.md

```diff
@@ -0,0 +1,64 @@
+# Member areas of interest
+
+OpenFeature technical committee members, maintainers, and approvers
+may list their areas of interest below, to help the community to find
+good points of contact for topics of interest across the project.
+
+Technical committee members are required to list these areas of
+interest, and this is optional for maintainers and approvers.  These
+listings may be useful to find an appropriate person to tag on an
+issue, for example.
+
+Members who are specifically employed by a company to contribute to
+the OpenFeature project are recommended to list their company
+affiliation, so that they may be contacted with vendor-specific
+concerns.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+<!-- tocstop -->
+
+## Technical committee members
+
+### [Dan O’Brien](https://github.com/InTheCloudDan), LaunchDarkly
+
+- 
+### [Todd Baert](https://github.com/toddbaert), Dynatrace
+
+- 
+### [Steve Arch](https://github.com/agentgonzo), CloudBees
+
+- 
+
+## Maintainers and approvers
+
+Maintainers and approvers are invited to list their areas of interest
+to further assist the community in finding appropriate points of
+contact.
+
+### [Alois Reitbauer](https://github.com/aloisreitbauer), Dynatrace
+
+- 
+### [Ben Rometsch](https://github.com/dabeeeenster), Flagsmith
+
+- 
+### [Justin Abrahms](https://github.com/justinabrahms), eBay
+
+- 
+### [Kevin Chu](https://github.com/kbychu), GitLab
```

Comment:
Kevin has left the board, I'm not sure if he should be mentioned here or not.

Reply:
Yes, he should be removed and Alex should be added.